components/predict.yaml

```diff
@@ -36,7 +39,21 @@ implementation:
         dataset_subfolder=
       fi
 
-      aws s3 sync --no-progress --exclude "synthetic*.txt" --exclude "*bootleg*" ${s3_input_datadir}${dataset_subfolder} dataset/almond
+      case $task_name in
+        *"multilingual"*)
+            dataset_dir=dataset/almond/multilingual
+        ;;
+
+        "almond"*)
+          dataset_dir=dataset/almond
+        ;;
+
+        *)
+           dataset_dir=dataset
```

Comment:
Does this mean prediction pipeline is fully compatible with non-almond tasks in genienlp?

Reply:
I've only tested it for NER datasets I added. Other tasks don't even run in genienlp yet and need to be updated.
On another topic, I think it would be a good idea to remove subdirectories such as  `almond` and `almond/multilingual` in genienlp and just pass the right dataset folder. And then we can remove these case statements everywhere in pipelines. What do you think? @s-jse @gcampax

Reply:
Do you mean removing `almond` and `almond/multilingual` everywhere in genienlp? How would we handle multi-task training (when we need it)?

Reply:
And you would still need case statements for downloading `almond` and `almond_dialogue_nlu` because they are in different S3 directories.

Reply:
Yeah if we want to do multi-tasking then we have to keep `almond` dir.
An alternate approach is to just check for almond vs non-almond dataset. Then we will sync the files from s3_data_dir to the correct directory indicated by task_name:
```
aws s3 sync --no-progress ${s3_input_datadir}${dataset_subfolder} dataset/$task_nmae 
```
except that all almond tasks will sync to dataset/almond regardless of the actual task name. we can change this in genienlp but I don't think we wanna go that route.
Then only `almond_multilingual` task needs to be changed in genienlp `(f'almond/multilingual/{dir}' --> f'almond/{dir}')`.  Other almond tasks will map correctly.

Reply:
I'm not sure what is being proposed here?

Reply:
Basically no more pattern checks for `almond/multilingual` in the pipelines. Just an if-else to check almond vs non-almond datasets.

Reply:
Is there any advantage to that? You still have a branch for almond vs the rest...

Reply:
It just makes the code simpler cause almond_multilingual task doesn't really need extra care. It has the same data structure as other almond tasks.